src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/resource/PrisonersInPrisonResourceTest.kt

```diff
@@ -5,12 +5,14 @@ import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.DisplayName
 import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
+import org.springframework.test.context.TestPropertySource
 import org.springframework.web.reactive.function.BodyInserters
 import uk.gov.justice.digital.hmpps.prisonersearch.PrisonerBuilder
 import uk.gov.justice.digital.hmpps.prisonersearch.QueueIntegrationTest
 import uk.gov.justice.digital.hmpps.prisonersearch.model.RestResponsePage
 import uk.gov.justice.digital.hmpps.prisonersearch.services.dto.PrisonersInPrisonRequest
 
+@TestPropertySource(properties = [ "index.page-size=1000" ])
```

Comment:
Optimisation when using the data loader - just reindex all prisoner records in one page